tools/cmake/RealmCoreConfig.cmake.in

```diff
@@ -1,28 +1,33 @@
 include(CMakeFindDependencyMacro)
 
 if(UNIX AND NOT APPLE)
-    if(NOT OpenSSL_DIR)
-        if(NOT EXISTS ${CMAKE_BINARY_DIR}/openssl/lib/cmake/OpenSSL/OpenSSLConfig.cmake)
-            set(OPENSSL_URL "http://static.realm.io/downloads/openssl/@OPENSSL_VERSION@/Linux/x86_64/openssl.tgz")
-            if(ANDROID)
-                set(OPENSSL_URL "http://static.realm.io/downloads/openssl/@OPENSSL_VERSION@/Android/${CMAKE_ANDROID_ARCH_ABI}/openssl.tgz")
+    if(REALM_USE_SYSTEM_OPENSSL)
```

Comment:
This was the same change as in the main CMakeLists.txt file, but this one is for projects like realm-sync that consume the generated RealmCore.cmake file.